week-1/J-functions/exercise2.js

```diff
@@ -1,4 +1,5 @@
 function triple(number) {
+  return number * 2;
```

Comment:
Double check the function name. Do the name and behaviour match?

Reply:
lool sorry so true